content/post/roadmap.md

```diff
@@ -0,0 +1,60 @@
+ 
+### Identify audience for such a document
+
+Ideally everyone in the organization has access to this road map. My preferred medium for communicating my road map is presentation. I can inject visuals if needed, color code where appropriate and use the natural aspect of "one topic per slide" to help navigate. 
+
+While everyone should have access, you don't need to make it digestible to everyone in the organization. You key audience should be leadership, relevant cross-functional partners and your team. That will help guide the level of detail required. I also like to present my road map to stakeholders, which allows me to keep it high level in the presentation and speak to it when necessary.
+
+### Relate to organization’s mission and values
+
+As a data team its incumbent upon us to ensure that decisions are tied to organizational objectives. This is part of thinking holistically about your business. In that vein, you should strive to tie each of your high level strategic goals to the organization's mission and/or objectives. It's a helpful exercise to tie each project, however indirectly, to the company's strategic objectives, even if you don't end up documenting that in the final road map. 
+
+You may end up tying your work back to, say, 2 of your organization's 3 strategic goals for the year. That's not in and of itself bad, but should be called out. This exercise reduces blind spots and forces you to think about how you can support all aspects of the mission.
+
+### Set up the scaffolding
+
+How do you start? After outlining our Data team mission and values (distinct from the broader company) I like to make explicit what areas of focus we're responsible for. For us, that's Data infrastructure, BI, analytics and strategic projects, Data Science and Clinical Data products. This helps to compartmentalize projects, especially those that are cross functional.
+
+You may find it more natural to split your road map by the various teams you support, e.g. Marketing, Product, Sales, etc... I prefer the former because it captures both technical differences that are relevant to the team (e.g. data analysts will skew heavily towards BI, data engineers towards data infrastructure, etc...) and "Analytics and Strategic projects" encapsulate projects that are cross functional (e.g. Marketing, Sales, Finance).
+
+### Sync with functional leaders 
+
+After putting down some projects that fit into the relevant areas of focus, set up time with key functional leaders and get their input. Hopefully you sequence this exercise at a time in which they are also thinking about their goals for the year. Have them help you fill in blind spots and loosely prioritize their projects. You may know of 4 Operations projects that are critical for 2019, but the COO may have a different idea on what the sequencing should be. Don't get bogged down by timing at this point, but getting the order correct is very helpful.
```

Comment:
> getting the order correct

Does this refer to chronology or priority?

Reply:
both? Tried to clarify.